### ISSUE TITLE
TypeScript definition

### ISSUE THREAD

**Issue by blakeembrey** [jackbowman/email-addresses#19]
If you're providing a TypeScript definition, can you provide it in external module format? It would allow NPM users to start using it without configuration. See http://www.typescriptlang.org/docs/handbook/typings-for-npm-packages.html.

**Comment by jackbowman** [comment]
@blakeembrey I can do that if it will be useful. I'm not a ts user, and I've only accepted a ts definition as a PR. More PRs welcome. Since I don't know ts I don't know whether the existing internal module definition is preferable to the external module definition.

It sounds like you want the `declare module...` line to be removed, so that the definition just exports functions, and then the package.json can include `"typings": "./lib/email-addresses.d.ts"`. Is that correct?

**Comment by blakeembrey** [comment]
Yes, absolutely correct. If it's an issue, I can submit a PR to correct it for you 😄 I don't actually use the module myself, just wanted to help out with a common mistake I see around.
